workflow/apm/database.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: APM Dashboard - Databases
```

Comment:
Recommend calling this just "Databases" because there's the red "APM" title above it.